packages/materialdesign-components/src/Button/Button.svelte

```diff
@@ -0,0 +1,60 @@
+<script>
+  import { setContext, getContext } from "svelte";
+  import Icon from "../Icon.svelte";
+  import ripple from "../Ripple.js";
+  import ClassBuilder from "../ClassBuilder.js";
+
+  const cb = new ClassBuilder("button", ["primary", "medium"]);
+
+  export let variant = "raised";
+  export let colour = "primary";
+  export let size = "medium";
+
+  export let href = "";
+  export let icon = "";
+  export let trailingIcon = false;
+  export let fullBleed = false;
+
+  export let text = "";
+  export let disabled = false;
+
+  $: if (!!icon) {
+    setContext("BBMD:icon:context", "button");
+  }
+
+  $: renderLeadingIcon = !!icon && !trailingIcon;
```

Comment:
Same as above

Reply:
![image](https://user-images.githubusercontent.com/36074859/73690672-cb833a80-46c8-11ea-9acf-018923a577a0.png)
This is true for the if statement in your first comment, but not this one. `renderLeadingIcon` is coerced to an empty string without the `!!icon` because thats what icon defaults to (see image).